addons/mail/models/discuss/discuss_channel_rtc_session.py

```diff
@@ -98,7 +115,7 @@ def _notify_peers(self, notifications):
     def _mail_rtc_session_format(self, fields=None):
         self.ensure_one()
         if not fields:
-            fields = {'id': True, 'channelMember': {'id': True, 'channel': {}, 'persona': {'partner': {'id', 'name', 'im_status'}, 'guest': {'id', 'name', 'im_status'}}}, 'isCameraOn': True, 'isDeaf': True, 'isSelfMuted': True, 'isScreenSharingOn': True}
+            fields = {'id': True, 'channelMember': {'id': True, 'channel': {}, 'persona': {'partner': {'id', 'name', 'im_status'}, 'guest': {'id', 'name', 'im_status'}}}}
```

Comment:
Does it actually hurt to return these fields when not necessary?

Reply:
yes the issue is that these fields are not updated in real time, so they are not as accurate as the client knowledge when the client is in a call where they can get real-time update through the rtc connection.